wrappers/csharp/cs-tutorial-8-D400-on-chip-calibration/Window.xaml.cs

```diff
@@ -0,0 +1,178 @@
+                    Console.WriteLine("    Serial number: {0}", dev.Info[CameraInfo.SerialNumber]);
+                    Console.WriteLine("    Firmware version: {0}", dev.Info[CameraInfo.FirmwareVersion]);
+
+                    var sensors = dev.QuerySensors();
+
+                    var depthProfile = sensors
+                                            .SelectMany(s => s.StreamProfiles)
+                                            .Where(sp => sp.Stream == Stream.Depth)
+                                            .Select(sp => sp.As<VideoStreamProfile>())
+                                            .OrderBy(p => p.Framerate)
+                                            .First();
+
+                    var colorProfile = sensors
+                                            .SelectMany(s => s.StreamProfiles)
+                                            .Where(sp => sp.Stream == Stream.Color)
+                                            .Select(sp => sp.As<VideoStreamProfile>())
+                                            .OrderBy(p => p.Framerate)
+                                            .First();
+
+
+                    cfg.EnableDevice(dev.Info[CameraInfo.SerialNumber]);
+                    cfg.EnableStream(Stream.Depth, depthProfile.Width, depthProfile.Height, depthProfile.Format, depthProfile.Framerate);
+                    cfg.EnableStream(Stream.Color, colorProfile.Width, colorProfile.Height, colorProfile.Format, colorProfile.Framerate);
+
+
+                    var pp = pipeline.Start(cfg);
+
+                    SetupWindow(pp, out updateDepth, out updateColor);
+                }
+
+                // Rendering task
+                var renderingPause = false;
+                var rendering = Task.Factory.StartNew(() =>
+                {
+                    while (!tokenSource.Token.IsCancellationRequested)
+                    {
+                        if (renderingPause) continue; //pause the rendering
+
+                        // We wait for the next available FrameSet and using it as a releaser object that would track
+                        // all newly allocated .NET frames, and ensure deterministic finalization
+                        // at the end of scope. 
+                        using (var frames = pipeline.WaitForFrames())
+                        {
+                            var colorFrame = frames.ColorFrame.DisposeWith(frames);
+                            var depthFrame = frames.DepthFrame.DisposeWith(frames);
+
+                            // We colorize the depth frame for visualization purposes
+                            var colorizedDepth = colorizer.Process<VideoFrame>(depthFrame).DisposeWith(frames);
+
+                            // Render the frames.
+                            if (colorFrame != null)
+                                Dispatcher.Invoke(DispatcherPriority.Render, updateDepth, colorizedDepth);
```

Comment:
Params for depth have been used if the color frame if statement - and vice-versa

Reply:
This convention is used cross-examples